themes/default/content/blog/retainondelete/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Retain on Delete"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-02-17T10:14:31Z
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting your post for review.
+draft: true
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: Introducing the new resource option retainOnDelete, to allow sharing of cloud resources.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - fraser-waters
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+While Pulumi lets you manage your infrastrucutre with code there are some cases where operations need to
+happen outside the standard deployment lifecycle. Pulumi now supports retaining resources on delete allowing a
+number of new workflows.
+
+<!--more-->
+
+We've seen a number of user questions over the years about keeping resources after pulumi deployments, or
+sharing resources between deployments. All of these worked down to needing a new [resource
+option](https://github.com/pulumi/pulumi/issues/7747) to tell the Pulumi engine to not actually delete a
+resource when replacing or removing it from the stack.
+
+## Retain on delete
+
+We now support a new resource option [retain on delete]({{< relref "docs/intro/concepts/resources/options/retainOnDelete" >}})
+that allows you to specifiy this case to the engine.
+
+When a resource marked with `retainOnDelete` is deleted or replaced by a Pulumi update the engine will not
+call through to the resource providers `Delete` function. This is the similar to AWS Cloud Formation's [Retain
+DeletionPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html).
+
```

Comment:
Consider adding a link sentence explaining that the following few sections are going to be about scenario examples.